net-misc/anydesk/anydesk-5.4.1.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit desktop systemd xdg-utils
+
+DESCRIPTION="Feature rich multi-platform remote desktop application"
+HOMEPAGE="https://anydesk.com"
+SRC_URI="amd64? ( https://download.anydesk.com/linux/${P}-amd64.tar.gz )
```

Comment:
This is not available to download anymore. Could you update, and also check their license if they allow re-distributing the package in our mirrors?

Reply:
Right.

So they move the downloads as new versions become available, thus, according to ebuild(5):

       SRC_URI = "https://example.com/path/${P}.tar.gz"
              Contains  a  list  of  URIs for the required source files.  **It can contain multiple
              URIs for a single source file**.  The list is processed in order if the file was  not

As such I've just taken to providing both possible URLs.  Hopefully this is done correctly as I've not ever needed this before (it tests out on my laptop).

Bumped to 5.5.0 either way, which uses the "current release" download scheme.

Regarding license, I'm not a lawyer, so my interpretation of the license is that it's unclear whether or not we're allowed to mirror or not.  So with a hard-hat on my recommendation is to play it safe and rather error on the side of not mirroring.  I'm not seeing anything specifically allowing us to mirror it, nor specifically preclude us from, but there are hints that they expect users to download the software from their own site.